test_fms/mpp/Makefile.am

```diff
@@ -50,21 +51,24 @@ test_mpp_print_memuse_stats_stderr_SOURCES=test_mpp_print_memuse_stats_stderr.F9
 test_mpp_print_memuse_stats_file_SOURCES=test_mpp_print_memuse_stats_file.F90
 test_mpp_memutils_begin_2x_SOURCES=test_mpp_memutils_begin_2x.F90
 test_mpp_memutils_end_before_begin_SOURCES=test_mpp_memutils_end_before_begin.F90
+test_read_input_nml_SOURCES=test_read_input_nml.F90
 
 # Run the test programs.
 TESTS = test_mpp_domains2.sh \
   test_mpp_pset2.sh \
   test_mpp2.sh \
   test_mpp_memuse \
   test_mpp_mem_dump \
-  test_mpp_memutils_mod.sh
+  test_mpp_memutils_mod.sh \
+  test_read_input_nml2.sh
 
 # These files will also be included in the distribution.
 EXTRA_DIST = input_base.nml \
   test_mpp_domains2.sh \
   test_mpp_pset2.sh	\
   test_mpp2.sh \
-  test_mpp_memutils_mod.sh
+  test_mpp_memutils_mod.sh \
+  test_read_input_nml2.sh
```

Comment:
You need to add `test_read_input_nml.sh` here

To get rid of the `./test_read_input_nml.sh: not found` error